sling/nlp/parser/trainer/spec.py

```diff
@@ -298,7 +298,7 @@ def build(self, commons, corpora):
     # For compatibility with DRAGNN, suffixes don't have an OOV item.
     self.commons = commons
     self.words = Lexicon(self.words_normalize_digits)
-    self.suffix = Lexicon(normalize_digits=False, oov_item=None)
+    self.suffix = Lexicon(self.words_normalize_digits, oov_item=None)
```

Comment:
Another TODO (for me): Make sure that the suffix feature here is the same as the new implementation.

Reply:
The C++ affix table still supports the case where we do not have the empty affix, but I think we could simplify it a bit if we could always assume that it was there.